src/BugsnagServiceProvider.php

```diff
@@ -439,6 +440,27 @@ protected function setupCallbacks(Client $client, Container $app, array $config)
                 }
             }));
         }
+
+        // Laravel 11 has a 'Context' class for storing metadata
+        // https://laravel.com/docs/11.x/context
+        if (class_exists(Context::class)) {
+            $client->registerCallback(function (Report $report) {
+                $context = Context::all();
+
+                $report->setMetaData(['Laravel Context' => $context]);
+            });
+
+            // only attach hidden context if enabled, otherwise sensitive data
+            // could leak to bugsnag
+            if (isset($config['attach_hidden_context']) && $config['attach_hidden_context']) {
+                $client->registerCallback(function (Report $report) {
+                    $hiddenContext = Context::allHidden();
+
+                    $report->setMetaData(['Laravel Hidden Context' => $hiddenContext]);
```

Comment:
Do you think it's important to know whether the context was hidden or not when they view the data? Just wondering if it would be better to combine the data in one tab/section?

Scanning the docs, it looks like the keys could clash for this though?